parts/linux/cloud-init/artifacts/manifest.json

```diff
@@ -21,6 +21,32 @@
             "default": "1.1.12"
         }
     },
+    "kube-proxy": {
```

Comment:
why do we need to define the metadata in manifest.json and manifest.cue ? shouldnt this be in a single place ? and the `.json` file gets generated at build time ?

Reply:
I believe this is the current behavior with `make generate`.  If we are going to use .cue file only for validation, I believe this will be addressed.